cas-management-webapp-support/src/main/java/org/jasig/cas/config/CasManagementWebAppConfiguration.java

```diff
@@ -43,6 +43,7 @@
  * @since 4.3.0
  */
 @Configuration("casManagementWebAppConfiguration")
+@Lazy(true)
```

Comment:
At least for dev; i.e. you want to ensure context loads correctly but you don't necessarily need to hit the web. As long as it loads, ...